app/assets/javascripts/L.GeoJSONvtTiles.js

```diff
@@ -369,6 +371,17 @@ L.GeoJsonvtTiles =  L.TileLayer.Canvas.extend({
     }
   },
 
+  prepareFeatureData: function(data) {
+      data.features.forEach(function(feature) {
+      var properties = feature.properties || feature.tags;
+      properties.color_cache = {};
+      var all_tags = [];
+      properties.snippets.forEach(function(snippet) {
+        all_tags = all_tags.concat(snippet.tags);
+      });
+      properties.all_tags = all_tags;
+    });
+  },
```

Comment:
and this part should be passed to the [L.geoJsonVtTiles caller](https://github.com/ondrej-hosak/las_control/blob/0900da4d826d283e05d1646c46058e62ae2b5fef/app/assets/javascripts/map.js#L334)